apps/shared/scripts/generate-gallery-tokens.mjs

```
@@ -0,0 +1,381 @@
+function main() {
+	// The resolution reads the shell package's SOURCE (styles.ts and the
+	// surface component .tsx files). Standalone app repos vendor only the
+	// COMPILED shell (.rocketride/shell), so a faithful regeneration — and
+	// therefore drift-checking — is only possible in the monorepo. Keep the
+	// committed map as-is and skip here.
+	if (!existsSync(STYLES_FILE)) {
+		console.log('shell source not present (standalone repo) - keeping committed tokenUsage.generated.ts');
+		return;
+	}
+	const rendered = renderModule(buildUsageMap());
+	if (process.argv.includes('--check')) {
```

ESLint

[error] 'process' is not defined.

(no-undef)